### PR TITLE
Improve setup-ci command with helper links and allow override

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: src/Console/Command/SetupCICommand.php [hunk 1]
```diff
@@ -42,29 +42,62 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+        if ($currentRepository === null) {
+            $this->symfonyStyle->error('Current repository name could not be resolved');
 
-                return self::FAILURE;
-            }
+            return self::FAILURE;
+        }
 
-            $workflowTemplate = FileSystem::read(__DIR__ . '/../../../templates/rector-github-action-check.yaml');
+        $workflowTemplate = FileSystem::read(__DIR__ . '/../../../templates/rector-github-action-check.yaml');
```
Comment (gplanchat) [inline src/Console/Command/SetupCICommand.php:72]: **🎯 suggestion:** I usually prefer this notation for readability

```suggestion
        $workflowTemplate = FileSystem::read(dirname(__DIR__, 3) . '/templates/rector-github-action-check.yaml');
```

File: src/Console/Command/SetupCICommand.php [hunk 2]
```diff
@@ -42,29 +42,62 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+
+        $this->symfonyStyle->note('The "rector.yaml" workflow was added');
+        $this->symfonyStyle->newLine();
+
+        $this->symfonyStyle->title('2 steps more to run you Github Action:');
+        $this->symfonyStyle->writeln(
+            '1) Generate new Github Token here:' .
+            PHP_EOL .
+            'https://github.com/settings/tokens/new'
+        );
```
Comment (gplanchat) [inline src/Console/Command/SetupCICommand.php:90]: **🎯 suggestion:** I would add a clickable link, in order to simplify usage

```suggestion
        $this->symfonyStyle->writeln(
            '1) Generate new Github Token here:' .
            PHP_EOL .
            '<href=https://github.com/settings/tokens/new>https://github.com/settings/tokens/new</>'
        );
```

File: src/Console/Command/SetupCICommand.php [hunk 3]
```diff
@@ -42,29 +42,62 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+
+        $this->symfonyStyle->newLine();
+
+        $this->symfonyStyle->writeln(
+            '2) Add it to your repository secrets under "GITHUB_TOKE" name:'
+            . PHP_EOL .
+           sprintf('https://github.com/%s/settings/secrets/actions/new', $currentRepository)
+        );
```
Comment (gplanchat) [inline src/Console/Command/SetupCICommand.php:98]: **🎯 suggestion:** Here too, I would add a clickable link, in order to simplify usage
‏
```suggestion
        $this->symfonyStyle->writeln(
            '2) Add it to your repository secrets under "GITHUB_TOKE" name:'
            . PHP_EOL .
            sprintf('<href=https://github.com/%s/settings/secrets/actions/new>https://github.com/%s/settings/secrets/actions/new</>‏', $currentRepository)
        );
```

File: src/Console/Command/SetupCICommand.php [hunk 4]
```diff
@@ -42,29 +42,62 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+
+        $this->symfonyStyle->newLine();
```
Comment (gplanchat) [inline src/Console/Command/SetupCICommand.php:100]: **💭 thought:** If it is possible to add some links to a documentation page:
* Fine grained Organization access token: https://docs.github.com/en/organizations/managing-programmatic-access-to-your-organization/setting-a-personal-access-token-policy-for-your-organization
* Fine grained user access token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-fine-grained-personal-access-token

**Review by gplanchat — COMMENTED** [review]
**👏 praise:** It's a great addition‏

File: src/Console/Command/SetupCICommand.php (hunk 1 above)
Comment (TomasVotruba) [reply src/Console/Command/SetupCICommand.php:72]: This actually shuts the PHPStan check for missing file.
Also is dynamic over static value.

File: src/Console/Command/SetupCICommand.php (hunk 3 above)
Comment (TomasVotruba) [reply src/Console/Command/SetupCICommand.php:98]: I see. How does this look in the Symfony style?

File: src/Console/Command/SetupCICommand.php (hunk 4 above)
Comment (TomasVotruba) [reply src/Console/Command/SetupCICommand.php:100]: I see. Most people don't read these, so we have limit for 2 links maximum. This command should help people just dump it and use it.

Which 2 links would you go for having typical Rector developer in mind?

**Comment by TomasVotruba** [comment]
@gplanchat Thanks for quick testing and feedback :+1:   This will allow people to use Rector in CI on massive level :clap:

File: src/Console/Command/SetupCICommand.php (hunk 3 above)
Comment (gplanchat) [reply src/Console/Command/SetupCICommand.php:98]: It depends on your shell, usually it prints the link bolder or sharper, and underlined.

See https://symfony.com/blog/new-in-symfony-4-3-console-hyperlinks

File: src/Console/Command/SetupCICommand.php (hunk 2 above)
Comment (TomasVotruba) [reply src/Console/Command/SetupCICommand.php:90]: See https://github.com/rectorphp/rector-src/pull/3457 :)

Comment (TomasVotruba) [reply src/Console/Command/SetupCICommand.php:90]: See https://github.com/rectorphp/rector-src/pull/3457 :)

File: src/Console/Command/SetupCICommand.php (hunk 4 above)
Comment (gplanchat) [reply src/Console/Command/SetupCICommand.php:100]: I think the best would be an info message at the end, telling "If you want to get deeper and configure actions for a whole organization, please see [...]" with the link to the organization access token configuration.

**Pull request merged**